src/vacuum-card.js

```diff
@@ -194,6 +218,25 @@ class VacuumCard extends LitElement {
     };
   }
 
+  renderHeaderAction() {
+    if (this.showHeaderAction == false) {
+      return html``;
+    }
+
+    const { header_action = [] } = this.config;
+
+    const execute = () => {
+      const [domain, name] = header_action.service.split('.');
+      this.hass.callService(domain, name, header_action.service_data);
+    };
+
+    return html`<div class="header_action"><ha-icon-button
```

Comment:
`ha-icon-button` is changed in the latest HA. Please define icons with `ha-icon`.

Reply:
Hi @denysdovhan.
You don't intend to fit the card to the latest version of HA?